### ISSUE TITLE
only display top-level collections for "browse collections"

### ISSUE THREAD

**Issue by rococodogs** [LafayetteCollegeLibraries/spot#613]
this should probably be do-able with an adjustment of the search builder.

**Comment by rococodogs** [comment]
so the solr query to get what we're looking for is: `has_model_ssim:Collection AND -member_of_collection_ids_ssim`, but haven't yet found how to translate that into a querystring.

it might worthwhile to revisit #282 and build out a collections landing page

**Comment by rococodogs** [comment]
this is in the [pull request], but just wanted to follow up from the previous comment with the answer:

```json
{
  "facet": true,
  "fq": ["has_model_ssim:Collection", "-member_of_collection_ids_ssim:[* TO *]"]
}
```

[pull request]: https://github.com/LafayetteCollegeLibraries/spot/pull/638

**Comment by rococodogs** [comment]
closed with #645
